src/openzaak/components/catalogi/openapi.yaml

```diff
@@ -5887,7 +5887,7 @@ paths:
         format: uuid
   /zaaktype-informatieobjecttypen:
     get:
-      operationId: zaakinformatieobjecttype_list
+      operationId: zaaktypeinformatieobjecttype_list
```

Comment:
that's a breaking change - clients operating on `operationId` (of which gemma-zds-clients is one) will crash on this change.

The schema introspection has a `get_operation_id` method, probably we should override it to keep the old one here.